django/contrib/admin/templatetags/admin_list.py

```diff
@@ -259,8 +259,7 @@ def link_in_col(is_first, field_name, cl):
                     '<a href="{}"{}>{}</a>',
                     url,
                     format_html(
-                        ' onclick="opener.dismissRelatedLookupPopup(window, '
-                        '&#39;{}&#39;); return false;"', result_id
+                        ' data-popup-opener="{}"', result_id
```

Comment:
line 257 is the problem line but you can't comment on a line that's not been changes